problems/countNumbers.js

```diff
@@ -1,17 +1,23 @@
-
-/** 
+/**
  * Count all the numbers in an array
- * 
+ *
  * @param {number[]} arr - An array containing numbers
  * @returns {object} - an object where the key is the number, and the value is the count of that number
- * 
- * ex: countNumbers([1,1,1,2,2,3,4]) 
+ *
+ * ex: countNumbers([1,1,1,2,2,3,4])
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+  let numbers = {};
+  for (i = 0; i < arr.length; i++) {
+    if (numbers[arr[i]] !== undefined) {
```

Comment:
if `numbers[arr[i]] `doesn't return `undefined` then the if statement resolves it to truthy so theres no need to compare it to undefined. This is still good though because it makes it easier for humans to read